packages/components/container/src/container.vue

```diff
@@ -3,43 +3,39 @@
     <slot />
   </section>
 </template>
-<script lang="ts">
-import { computed, defineComponent } from 'vue'
+<script lang="ts" setup>
+import { computed, useSlots } from 'vue'
 import { useNamespace } from '@element-plus/hooks'
 
 import type { Component, VNode } from 'vue'
 
-export default defineComponent({
+defineOptions({
   name: 'ElContainer',
-  props: {
-    direction: {
-      type: String,
-      default: '',
-    },
+})
+const props = defineProps({
+  direction: {
+    type: String,
+    default: '',
```

Comment:
**Nitpick:** remove unnecessary `default: ''` since this is a `String` type.

Reply:
Done  🏓